### PR TITLE
#88 Text info summary (and Default Summary View improvements)

### PR DESCRIPTION
**Pull request by CarlosNZ**
Text info now has a Summary View and it appears on the Summary and Review pages.

Had a problem getting custom Summary Views to work, and after discussing and resolving with @andreievg the best solution was to ensure each plugin has its own SummaryView.tsx file. However we can still utilise the "DefaultSummaryView" component by passing it down to each plugin and it just returns the DefaultSummary component.

Something to think about: Now info-type elements always appear in Summary and Review pages. Would we *always* want this? Do you think we need another boolean field in `template_element` called `show_in_summary`, which would default to `true` but can be turned off sometimes.

### CODE REVIEW

File: src/components/Review/ReviewSection.tsx [hunk 1]
```diff
@@ -88,6 +88,7 @@ const ReviewSection: React.FC<ReviewSectionProps> = ({
         </Accordion.Title>
         <Accordion.Content active={isOpen}>
           {Object.entries(pages).map(([pageName, elements]) => {
+            console.log('elements', elements)
```
Comment (nmadruga) [inline src/components/Review/ReviewSection.tsx:91]: Please remove console

File: src/components/Review/ReviewSection.tsx [hunk 2]
```diff
@@ -104,27 +105,53 @@ const ReviewSection: React.FC<ReviewSectionProps> = ({
...
+                                        color="blue"
+                                        style={{ minWidth: 100 }}
+                                        onClick={() => setDecisionArea(review, summaryViewProps)}
+                                      />
+                                    </Grid.Column>
+                                  </Grid.Row>
+                                </Grid>
+                              </Card.Content>
+                            </Card>
                           </Grid.Row>
```
Comment (nmadruga) [inline src/components/Review/ReviewSection.tsx:155]: Not sure why is showing all this diff here. Perhaps there is another PR included in this one too?

**Review by nmadruga — APPROVED** [review]
LGTM

File: src/components/Review/ReviewSection.tsx (hunk 2 above)
Comment (nmadruga) [reply src/components/Review/ReviewSection.tsx:155]: There is actually a compilation error happening due to this change :(

Comment (nmadruga) [reply src/components/Review/ReviewSection.tsx:155]: I think this is a conflict resolution mistake. It needs to be removed before merging!

Comment (CarlosNZ) [reply src/components/Review/ReviewSection.tsx:154]: Weird. The diff I can explain, not so much the compilation error. The reason for the diff is that this whole block used to be within `if (category === TemplateElementCategory.Question) {`, but I removed that as we now show all, not just question types, hence the indentation for the whole block changed.

Have fixed the error now. If you look at the changes with "Hide whitespace changes" selected, you'll see the diff is just the couple of lines at each of the block.

**Review by nmadruga — APPROVED** [review]
Great. Thanks!

**Pull request merged**